### PR TITLE
Feathers auth

### PR DESCRIPTION
**Pull request by askariya**

### CODE REVIEW

File: src/components/AccountModal.vue [hunk 1]
```diff
@@ -45,6 +45,27 @@ export default {
         {name: 'composition 4'}
       ]
     }
+  },
+  // declare all methods here
+  methods: {
+    logMe () {
+      console.log('Test Log from the methods declaration')
```
Comment (kusamkaur16) [inline src/components/AccountModal.vue]: is this just a place holder?

File: src/components/AccountModal.vue [hunk 2]
```diff
@@ -45,6 +45,27 @@ export default {
...
+    }
+  },
+  // this runs once per construction of this modal
+  created () {
+    // this.$services.compositions.create({
```
Comment (ckdoan) [inline src/components/AccountModal.vue]: remove comments

File: src/components/LoginModal.vue [hunk 3]
```diff
@@ -36,10 +46,113 @@
...
+    async login_user () {
+      document.getElementById('error-display-login').innerText = ''
+      const getCredentials = () => {
+        const user = {
+          // email: "feathers@example.com",
+          // password: "secret"
+          username: this.username,
+          password: this.password
+        }
+        console.log(user.username)
```
Comment (ckdoan) [inline src/components/LoginModal.vue]: do we want this printed out later on ?

File: src/components/LoginModal.vue [hunk 4]
```diff
@@ -36,10 +46,113 @@
...
+          } else {
+            // If we get login information, add the strategy we want to use for login
+            const payload = Object.assign({ strategy: 'local' }, credentials)
+
+            await this.$feathers.authenticate(payload)
+          }
+
+          // If successful, show the application UI
+          console.log('Show main application now')
+          // var modalDialog = document.getElementById('loginModal')
```
Comment (ckdoan) [inline src/components/LoginModal.vue]: remove comments

File: src/components/AccountModal.vue (hunk 1 above)
Comment (askariya) [reply src/components/AccountModal.vue]: For now, yeah. It's where I'm gonna add the logic for the "change username and password" functionality when I create my next branch,

File: src/components/RegisterModal.vue [hunk 5]
```diff
@@ -1,69 +1,244 @@
...
+      }
+    }
+    // agreement: function (value) {
+    //   return Validator.value(value).required()
+    // }
+  },
+
+  methods: {
+
+    // TODO Delete -- doesn't do what it should
```
Comment (ckdoan) [inline src/components/RegisterModal.vue]: delete

File: src/components/AccountModal.vue (hunk 2 above)
Comment (askariya) [reply src/components/AccountModal.vue]: Keeping this comment as it's a placeholder for what we need to do later once we can build compositions.

File: src/components/LoginModal.vue (hunk 3 above)
Comment (askariya) [reply src/components/LoginModal.vue]: No, I will comment them out, want to keep them for debugging the change username and password stuff later.

**Pull request merged**